### PR TITLE
Deadlock retry

### PR DESCRIPTION
**Pull request by kirill-levitskiy**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -184,7 +193,11 @@ which can read this file on the previous step and return value like this:
 ```
 and in this action you need put `query_string` (or some JSONata expression) to `Sql Injection string`:
 
-![image](https://user-images.githubusercontent.com/16806832/53736167-a02d1580-3e91-11e9-93dd-84c0048f72d2.png)
+![image](https://user-images.githubusercontent.com/40201204/61515864-470eea00-aa0c-11e9-837b-4cb24e88f512.png)
+
+#### Number of retries in case of deadlock transaction
+You can specify maximum number of retries, that is intended to help to solve lock's issues in case of a deadlock transaction.
+The delay between retries is 1 second.
```
Comment (shulkaolka) [inline README.md]: Please, add a description for default behavior

File: package.json [hunk 2]
```diff
@@ -23,23 +23,23 @@
...
+    "pg-cursor": "1.3.0",
+    "pg-promise": "8.6.4",
+    "q": "1.5.1",
     "sinon": "7.2.3",
     "sql-parser": "zubairov/sql-parser",
-    "sql-template-strings": "^2.2.2"
+    "sql-template-strings": "2.2.2"
   },
   "devDependencies": {
     "eslint": "^5.13.0",
```
Comment (shulkaolka) [inline package.json]: Please, delete `^`

File: spec/integration/action/executeSqlInjection.test.js [hunk 3]
```diff
@@ -41,4 +145,15 @@ describe('GeneralSqlQuery Action test', () => {
     await generalSqlQuery.process.call(emitter, msgWithError, cfg);
     expect(emitter.emit.calledWith('error')).to.be.equal(true);
   });
+
+  it('should be deadlock', async () => {
+    // for i in {1..3}; do psql elasticio_testdb postgres -c "select f_test('blah')"; done
```
Comment (shulkaolka) [inline spec/integration/action/executeSqlInjection.test.js]: Should it be deleted?

Comment (kirill-levitskiy) [reply spec/integration/action/executeSqlInjection.test.js]: I think, this note (shell command) will be useful for understanding the mechanism of the test

File: package.json (hunk 2 above)
Comment (kirill-levitskiy) [reply package.json]: Removed before, I have no clue why it appeared in PR ¯\_(ツ)_/¯

**Pull request merged**